### ISSUE TITLE
Edit function breaks link

### ISSUE THREAD

**Issue by tinyn** [RadhiFadlillah/shiori#94]
In server mode when you select edit

1. The tags come back without the comma separator
![screen shot 2018-05-25 at 5 32 36 am](https://user-images.githubusercontent.com/39618991/40537533-2be09456-5fdd-11e8-8a5a-74287f7f2a85.png)

2. If you edit any field an select "OK" the "link" to cache no longer exists. From the code, the issue is here

The target seems to be lost
Ex of broken link:
```
<a target="_blank" class="bookmark-link"><!----> <p class="title">http://main.lv/writeup/kernel_dev_hwrng.md</p> <p class="excerpt"></p></a>
```
Example of "good" link:
```
<a href="/bookmark/54" title="View cache" target="_blank" class="bookmark-link"><!----> <p class="title">DNSCrypt - ArchWiki</p> <p class="excerpt">DNSCrypt encrypts and authenticates DNS traffic between user and DNS resolver. While IP traffic itself is unchanged, it prevents local spoofing of DNS queries, ensuring DNS responses are sent by the server of choice. [1]</p></a>
```
Here is what the commandline shows post edit.
```
53. http://main.lv/writeup/kernel_dev_hwrng.md
    > http://main.lv/writeup/kernel_dev_hwrng.md
    # code, drivers, kenel, linux

54. DNSCrypt - ArchWiki (8-11 minutes)
    > https://wiki.archlinux.org/index.php/DNSCrypt
    + DNSCrypt encrypts and authenticates DNS traffic between user and DNS resolver. While IP traffic itself is unchanged, it prevents local spoofing of DNS queries, ensuring DNS responses are sent by the server of choice. [1]
    # admin, dns, net, security
```

EDIT: Further testing shows this does not happen for the DNSCrypt entry. It will drop the cache link when the edit is first made, but on refresh it returns. It appears rather that some sites will not cache (maybe?). I have deleted and added back the kernel_dev_hwrng entry but no cache link is formed. This may be the actual issue.

EDIT2: Looks like that is the case. From the DB
```
sqlite> select * from bookmark WHERE id=58;
58|http://main.lv/writeup/kernel_dev_hwrng.md|/dev/hwrng||Details on the HWRNG driver in the Linux kernel||0|0|2018-05-25 18:52:08

sqlite> select * from bookmark_content_content WHERE docid=58;
58|/dev/hwrng||
```

Not sure why other sites cached, but two of them now have not.

**Comment by RadhiFadlillah** [comment]
@tinyn thanks for the report. It should be fixed by [`65d6ddd`](https://github.com/RadhiFadlillah/shiori/commit/65d6ddd495aa88deec3dc602d28856c3ba8de555).

**Comment by RadhiFadlillah** [comment]
I’m closing this issue for now because :

- The issue has been fixed by [`65d6ddd`](https://github.com/RadhiFadlillah/shiori/commit/65d6ddd495aa88deec3dc602d28856c3ba8de555) and;
- It has been inactive for a while, so I guess this issue has been solved. 

Feels free to reopen it if needed.

Thank you.
